src/angularComponent/ngcOmniboxSuggestionsDirective.js

```diff
@@ -1,14 +1,53 @@
+import angular from 'angular';
 import NgcOmniboxSuggestionsController from './ngcOmniboxSuggestionsController.js';
 
-export default function ngcOmniboxSuggestionsDirective() {
+ngcOmniboxSuggestionsDirective.$inject = ['$templateCache'];
+export default function ngcOmniboxSuggestionsDirective($templateCache) {
   return {
     restrict: 'E',
     require: '^^ngcOmnibox',
     scope: true,
     controller: NgcOmniboxSuggestionsController,
     controllerAs: 'suggestions',
     compile(tElement) {
-      tElement.attr('role', 'listbox');
+      const el = tElement[0];
```

Comment:
If you're never going to use the jqLite wrapped `tElement`, maybe consider using a destructured array as the arg?

```
compile([tElement]) {
  // now "tElement" is just a pure HTMLElement
}
```